packages/tools/client-debugger/client-debugger-view/src/components/data-object-views/SharedStringView.tsx

```diff
@@ -23,7 +24,7 @@ export interface SharedStringViewProps {
 export function SharedStringView(props: SharedStringViewProps): React.ReactElement {
     const { sharedString } = props;
 
-    const [text, setText] = React.useState<string>(sharedString.getText());
+    const [_text, setText] = React.useState<string>(sharedString.getText());
```

Comment:
Do we need this state anymore? I think since we're just passing the SharedString to the CollaborativeTextArea component, we should be able to remove this.

Reply:
Removed state, only left `const { sharedString } = props;` inside the `SharedStringView` so that the `CollaborativeTextArea` can pass the `sharedString` into the `SharedStringHelper()`